lib/resolver.manifest.js

```diff
@@ -185,13 +188,21 @@ module.exports = class PodletClientManifestResolver {
                     );
                 }
 
+                if (manifest.value.assets.css) {
+                    outgoing.stream.emit('css', manifest.value.assets.css);
```

Comment:
We can not do this here. This will cause the css (and js) event to be emitted only when the manifest is fetched from a podlet. After its fetched, the manifest is cached in the client and one will never reach this point. 

We need to emit this here-ish: https://github.com/podium-lib/client/blob/master/lib/resolver.content.js#L258

Reply:
ah right of course